src/php/config/php.ini

```diff
@@ -1769,11 +1769,16 @@ opcache.enable=1
 ; Determines if Zend OPCache is enabled for the CLI version of PHP
 ;opcache.enable_cli=0
 
+# Set default values for: 
+# opcache.memory_consumption
+# opcache.interned_strings_buffer
+# See https://github.com/nextcloud-snap/nextcloud-snap/issues/2024
```

Comment:
No need to refer back to the issue. The commit/git history will do that for us.

Reply:
ok - do I just need to comment those out again? Will user input uncomment and set the new values?